nav2_costmap_2d/test/integration/test_costmap_2d_publisher.cpp

```diff
@@ -0,0 +1,156 @@
+//
```

Comment:
Remove this - its not going to be picked up as a proper copyright header. Only the stuff below is necessary